tests/unit/lib/middleware/instrument.ts

```diff
@@ -22,7 +22,7 @@ registerSuite('lib/middleware/instrument', function() {
 
   let server: Server;
   let shouldInstrumentFile: sinon.SinonStub<[string]>;
-  let instrumentCode: sinon.SinonStub<[string, string]>;
+  let instrumentCode: sinon.SinonStub<[string, string, (boolean | undefined)?]>;
```

Comment:
Isn't `(boolean | undefined)?` a bit redundant?

Reply:
Yes it is. I just used what sinon spit out at me in the error but I shortened it to `boolean?`